examples/android/app/src/main/proto/helloworld.proto

```diff
@@ -29,6 +29,7 @@
 
 syntax = "proto3";
 
+option optimize_for = LITE_RUNTIME;
```

Comment:
Oh, I know why. That's because the Lite API is basically the same. The protoc functions we call from our plugin would end up behaving the same even though they think we are using full proto (not lite).